### ISSUE TITLE
Unable to gather data regarding sync status using the prometheus pull consumer

### ISSUE THREAD

**Issue by wwsean08** [F5Networks/f5-telemetry-streaming#74]
### Is your feature request related to a problem? Please describe.
Using the prometheus pull consumer, it is currently not possible to get data related to sync status.  I have verified the data is available via the default pull consumer, the metric I care most about is the syncStatus.

### Describe the solution you'd like
The sync status (and potentially more data sync related data) exposed via a guage from the prometheus consumer.  Given that it would be a guage the value would have to be translated from "In Sync" to a number (perhaps 0), and a different number for when it's out of sync.  This would then allow showing the sync status on a dashboard in grafana using prometheus as the data source.

### Describe alternatives you've considered
It's possible I could find (or try and write) my own plugin for grabbing this one piece of data from the default pull consumer, however that seems a bit excessive.

### Additional context
Feel free to reach out to me via Teams to chat if you need more info about the request as we are currently using this internally at F5.

**Comment by dstokesf5** [comment]
Since the desired data is available in the default pull consumer, I am relabeling this as a bug. The Prometheus consumer should have the same data available as the default pull consumer.

**Comment by dstokesf5** [comment]
Turns out feature request was more accurate. The Prometheus consumer is metrics only, so it does not pick up the text based fields like syncStatus.

**Comment by wwsean08** [comment]
Hence my explanation of how I would Implement it.  I'll be back in the office next week if you'd like to discuss it @dstokesf5

**Comment by meghdivya** [comment]
Hi guys,

Any timeline for having a fix for this?

**Comment by megamattzilla** [comment]
@dstokesf5 has this feature request been added to the backlog? Any insights on where it's at on the sprint/roadmap would be appreciated. 

I am working with an F5 customer who has a large number of Big-IP devices using TS and Prometheus who is running into this issue.

**Comment by dstokesf5** [comment]
I have added AUTOTOOL-2330 to our internal product backlog to present sync information as a metric for the Prometheus consumer.

Since the below fields are filtered out for not being numbers, the current proposal is to create a new field (isSynchronized) for the Prometheus consumer that will be 1 if the device is in sync and 0 otherwise.
```json
        "syncMode": "standalone",
        "syncColor": "green",
        "syncStatus": "Standalone",
        "syncSummary": " ",
```

**Comment by dstokesf5** [comment]
This issue was resolved as part of the 1.20.0 release.
